environments/staging/deploys.yml

```diff
@@ -933,3 +933,51 @@ bare-template.aragonpm.eth:
       ipfsHash: QmbB5XPhn4wgF3zLYDBzbbwcx8yaVemwDZZzX6GJz8HMe4
       contractAddress: '0x6761aE15235f137daFBdA6C79af627697c61BDb5'
       commitHash: 72d9d0ccfd2c0a88eb2c4c664150471887d07b37
+aragon-fundraising.aragonpm.eth:
+  versions:
+    2.0.0:
+      date: 2019-12-12T11:33:30.000Z
+      txHash: '0xa3d11e9a2e179cb0f58bc4d0f3ded88ffd8b2339ef3ecd865d706e8a38221fc4'
+      ipfsHash: QmX6GNzSo3KWCWmhWWJEEpmU5n9ZranqzDwAJVPQKg2w4G
+      contractAddress: '0x58A325C03572CE11C054a5F19d2b9D7fc0F96490'
```

Comment:
The latest version `2.0.1` contract address is: `0xF7f519dDE49FB04A5492035Be9013811b1685e1d`. The current address is the one of `v1.0.0`.